templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+**Describe what this pull request brings**
+<!--
+Some examples:
+
+1. This pull request brings component Hero for the page Home.
+
+2. This pull request adds fixes for the responsive version of the page Blog.
+
+3. This pull request fixes user assignment via phone number in the existing stay.
+-->
+
+**Describe any additional changes**
+<!--
+An example:
+
+- fixes Header's height
+- introduces a new SASS variables $header-height
+- renames global CSS class "block" into "section"
+- fixes a typo in Hero for Home
+
+If this pull request does not contain any additional changes, feel free to remove this section.
+-->
+
+**Steps to test**
+<!--  Need a good example or template -->
```

Comment:
```
<! --
An example:

1. Pull the changes
2. Run `npm i`
3. Start the project in development mode, `npm run start`
4. Follow `/any-page`
5. Confirm, that everything is looks and works as expected
-->
```

Reply:
The problem that most time we have a deployed version so we can simply attach the link and that's it, there is no point to describe any other steps because they are obvious. I'm thinking where we could attach this link and can't think a good place for it.

But the example is good and can be used in a more complicated project / a project without branch deploys

Reply:
Well, in that case this is the exact place you want to attach this link, because exploring deployed version is a part of a test process 😉 

```
1. Open the *link*
2. Follow `/any-page`
3. Confirm that everything looks and works as expected
```